### DISCUSSION TITLE
Unable to purchase a domain

### DISCUSSION THREAD

**Discussion by lexonas** [vercel/community#4326] [Help] [answered]
### Summary

I am trying to buy a .lt domain, it is available, no issues. I select it, click buy, it processes, charges my card, then gives this error:

![image](https://github.com/vercel/community/assets/30417379/46939380-89bd-4fb0-9611-fff23d74a26f)

And then I get a refund and that is it. I tried it several times but with no luck. 

Any ideas?

### Steps to Reproduce

Select Domain- > buy -> find the wanted domain, select but with auto renew

**Comment by amyegan** [comment, marked as answer]
Hi @lexonas. Unfortunately, the issue seems to stem from the `.lt` TLD. Instead, you can buy your custom domain with another registrar and use DNS records or NS records to point the domain at your Vercel project.

https://vercel.com/docs/projects/domains/add-a-domain

**Reply by lexonas** [reply]
Yes, that worked! Thank you!
